Formula/curlcpp.rb

```diff
@@ -0,0 +1,67 @@
+class Curlcpp < Formula
+  desc "Object oriented C++ wrapper for CURL (libcurl)"
+  homepage "https://josephp91.github.io/curlcpp"
+  url "https://github.com/JosephP91/curlcpp/archive/refs/tags/1.5.tar.gz"
+  sha256 "d1c236be862965d5882a842be3112b16e7f49d6fdff9906f7aec71ee8669d60e"
+  license "MIT"
+
+  depends_on "cmake" => :build
+
+  uses_from_macos "curl"
+
+  on_linux do
+    depends_on "curl"
+  end
```

Comment:
```suggestion
```
This is the same as above

Reply:
Sorry, I don't see any differences between the two. What's the suggested change?

Reply:
Removing these three lines because it's exactly the same as `uses_from_macos "curl"`

Reply:
Right. Will fix it asap. Thanks!

Reply:
Done!

Reply:
@SMillerDev should be all fixed now